### DISCUSSION TITLE
Deployment is not working

### DISCUSSION THREAD

**Discussion by tekuto1437** [vercel/community#7088] [Help]
### Summary

I am having trouble with my project deployment, i have had several times Error NOT_FOUND, tried all possible solutions and nothing is helping me.

### Steps to Reproduce

https://2024-dp-43-deniss-jerosenko.vercel.app/

https://2024-dp-43-deniss-jerosenko-git-master-denis-projects-d575ee45.vercel.app/
https://2024-dp-43-deniss-jerosenko-rbp9056kv-denis-projects-d575ee45.vercel.app/
![image](https://github.com/vercel/community/assets/78407541/4a01855f-1849-4893-a26c-c4168931753e)

**Comment by pjhawksr** [comment]
I am seeing the same message for a new project I started yesterday. I can deploy to my local environment without issue.

**Reply by karlie-moyo** [reply]
I'm having the same issue as well...
